internal_testsuites/golang/testsuite/startosis_add_service_test/startosis_add_service_invalid_name_test.go

```diff
@@ -0,0 +1,57 @@
+package startosis_add_service_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/kurtosis-tech/kurtosis-cli/golang_internal_testsuite/test_helpers"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+const (
+	invalidServiceName                       = "this;.is:invalid"
+	addServiceWithInvalidServiceNameTestName = "add-service-invalid-service-name"
+	addServiceInvalidServiceNameTestScript   = `
+CONTAINER_IMAGE = "kurtosistech/example-datastore-server"
+GRPC_PORT = 1323
+
+def run(plan):
+	config = ServiceConfig(
+		image = CONTAINER_IMAGE,
+		cpu_allocation=500,
+		memory_allocation=256,
+		ports = {
+			"grpc": PortSpec(number = GRPC_PORT, transport_protocol = "TCP")
+		}
+	)
+	plan.add_service(service_name = "%s", config = config)
+`
+)
+
+func TestAddServiceWithInvalidServiceNameFailsValidation(t *testing.T) {
```

Comment:
added an integration test as I couldn't find a good spot to test Validation errors apart from this

Reply:
Yeah we don't have a good place to test validation mechanism right now... :(